woocommerce/pinterest-for-woocommerce
# Refresh catalog tab after settings change

### Changes proposed in this pull request

After settings are saved, refresh the catalog page, since it's content depends on the "enable product sync" setting.

The resolver cache will be invalidated, and logic will be retriggered.

#### Screenshots
Before:

![capture_YYYY08DD1908SS_before](https://user-images.githubusercontent.com/4016167/127894381-443dc1d5-34d8-4cee-a6da-4fdcdb992452.gif)

After:
![capture_YYYY08DD1908SS_after](https://user-images.githubusercontent.com/4016167/127894918-44282f1c-dbe0-424b-8ba2-9b02bd72cc0a.gif)


### Detailed test instructions

1. Go to the settings tab, change the "Enable product sync" setting. 
2. Go to the catalog tab
3. Verify that the information there is consistent with that setting. Eg. if product sync is disabled, You should see the "Go to settings to enable product sync" message. Otherwise you should see the status of the product sync as usual.
